script/build.ts

```diff
@@ -178,6 +181,12 @@ function copyStaticResources() {
   fs.copySync(common, destination, { clobber: false })
 }
 
+function copyLogo() {
+  const logoImage = path.join(projectRoot, 'app', 'static', 'logos', '512x512.png')
```

Comment:
We actually have a folder of dedicated Linux content at `app/static/linux/` that already gets copied across.

For the sake of simplicity I'd just copy this file into that location with the name you want it to be, rather than have a separate method that runs on all platforms.